app/views/widgets/shared/_omninav.html

```diff
@@ -302,7 +302,7 @@
                           <a href="https://blogs.chapman.edu/business/" title="Argyros School of Business &amp; Economics">Argyros School of Business &amp; Economics</a>
                         </li>
                         <li>
-                          <a href="https://blogs.chapman.edu/ces/" title="College of Educational Studies">College of Educational Studies</a>
+                          <a href="https://blogs.chapman.edu/ces/" title="Attallah College of Educational Studies">College of Educational Studies</a>
```

Comment:
@mandythomas Are we leaving the old name here because the name hasn't been changed for the blog?